docs/roadmap.md

```diff
@@ -0,0 +1,124 @@
+# CDF SIG Software Supply Chain Roadmap
+
+## Quick Links
+
+- [Overview](#overview)
+- [Charter](#charter)
+- [Outcomes](#outcomes)
+- [Roadmap](#roadmap)
+- [Contributors](#contributors)
+
+## Overview
+
+This document outlines the roadmap for the Continuous Delivery Foundation (CDF)
+Software Supply Chain Special Interest Group (SIG). The roadmap represents the
+initiatives of the open source collaborations. The roadmap does not specify
+timelines or delivery dates, but rather uses time horizons (now, next, later).
+
+## Charter
+
+The Software Supply Chain SIG aims to study different aspects of CI/CD systems, securing
```

Comment:
Suggest: "The Software Supply Chain SIG aims to study different aspects of CI/CD systems and share best practices for CICD with a focus on securing..."